src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -2065,6 +2065,18 @@ def migrate_sampletypes_to_dx(tool):
 
     logger.info("Convert SampleTypes to Dexterity [DONE]")
 
+@upgradestep(product, version)
```

Comment:
Remove this `upgradestep` decordator from here, cause what this function `reindex_getDueDate` function is doing is not likely to break other add-ons